### PR TITLE
[turnstile] Add changelog for October 28th

### PR DESCRIPTION
**Pull request by Le0Developer**
Relevant prs: #6346 

Not too sure about the links & the quick summary.

### CODE REVIEW

**Review by punkeel — APPROVED** [review]
thanks!

File: content/turnstile/changelog.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ weight: 10
 
 # Changelog
 
+## 2022-10-25
```
Comment (punkeel) [inline content/turnstile/changelog.md]: can please you change the date to be 2022-10-28?

Comment (Le0Developer) [reply content/turnstile/changelog.md]:
```suggestion
## 2022-10-28
````

**Review by migueldemoura — APPROVED** [review]

**Comment by Blackbeetle447** [comment]
Thanks

**Pull request merged**